### PR TITLE
Mz/fix linting issues

### PR DESCRIPTION
**Pull request by mszuyev**
# Related Github tickets

- https://github.com/VolumeFi/paloma/issues/9

# Background

Enabling static checks makes code better

# Testing completed

- [x] build, tests

# Breaking changes

- [x] I have checked my code for breaking changes

### CODE REVIEW

**Review by measure-fi — APPROVED** [review]

**Review by measure-fi — APPROVED** [review]

File: x/scheduler/keeper/keeper_setup_test.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 	tmdb "github.com/tendermint/tm-db"
 )
 
+//nolint:deadcode
 func newSchedulerKeeper(t testing.TB) (*Keeper, sdk.Context) {
```
Comment (taariq) [inline x/scheduler/keeper/keeper_setup_test.go]: We can remove the dead code. Let's definitely clean this up.

**Comment by taariq** [comment]
@mszuyev We are okay to REMOVE the dead code from this PR.

**Review by taariq — CHANGES_REQUESTED** [review]
Let's remove the unused code.

File: x/scheduler/keeper/keeper_setup_test.go (hunk 1 above)
Comment (mszuyev) [reply x/scheduler/keeper/keeper_setup_test.go]: Done in the commit https://github.com/palomachain/paloma/pull/755/commits/0d65fc0c9f5a307ac368632d2d9043624b547834

**Pull request merged**